cmake/targets/BuildBun.cmake

```diff
@@ -1260,6 +1260,7 @@ endif()
 
 if(WIN32)
   target_link_libraries(${bun} PRIVATE
+    advapi32
```

Comment:
what's advapi32?

Reply:
Advanced Windows 32 Base API

Reply:
We already most likely implicitly had it linked anyways, this is just a little bit more explicit 
<img width="946" height="363" alt="image" src="https://github.com/user-attachments/assets/3a762c4d-4356-40e6-8542-f8904042be15" />